.github/workflows/ee-nlc-snapshot-push.yml

```diff
@@ -85,6 +90,7 @@ jobs:
         run: |
           docker buildx build --push \
             --build-arg HZ_VERSION=${HZ_VERSION} \
+            --build-arg JDK_VERSION=${{ matrix.jdk }} \
             --build-arg HAZELCAST_ZIP_URL=${HAZELCAST_ZIP_URL} \
             --label hazelcast.ee.revision=${{ github.event.inputs.HZ_EE_REVISION }} \
             --tag ${NLC_IMAGE_NAME}:${HZ_VERSION} \
```

Comment:
You need modify tags to have `${NLC_IMAGE_NAME}-jdk${{ matrix.jdk }}:${HZ_VERSION}` for all JDK versions and additionally `${NLC_IMAGE_NAME}-:${HZ_VERSION}` for the default JDK

Reply:
Looks like I added this modification to the test build and not this one, will switch around :)